boardswarm/src/gpio.rs

```diff
@@ -45,6 +45,10 @@ pub async fn start_provider(name: String, parameters: serde_yaml::Value, server:
         (registry::PROVIDER, PROVIDER),
     ];
     let parameters: GpioParameters = serde_yaml::from_value(parameters).unwrap();
+    if parameters.match_.is_empty() {
+        warn!("matches is empty - could match random device");
```

Comment:
your commit message doesn't match your code. Also i suspect if you set your log level to warn it won't log the start_provider span (but that can be a seperate fix)

Reply:
Fixed that.